### PR TITLE
Add input size check to wc_ed448_import_public

### PR DESCRIPTION
**Pull request by guidovranken**
Additionally, match the code format of wc_ed25519_import_public.

### CODE REVIEW

**Comment by dgarske** [comment]
Jenkins @guidovranken is OK to test

File: wolfcrypt/src/ed448.c [hunk 1]
```diff
@@ -611,35 +611,36 @@ int wc_ed448_import_public(const byte* in, word32 inLen, ed448_key* key)
     int ret = 0;
 
     /* sanity check on arguments */
-    if ((in == NULL) || (key == NULL)) {
-        ret = BAD_FUNC_ARG;
+    if (in == NULL || key == NULL) {
+        return BAD_FUNC_ARG;
```
Comment (SparkiDev) [inline wolfcrypt/src/ed448.c:615]: Only one exit point per function

File: wolfcrypt/src/ed448.c [hunk 2]
```diff
@@ -611,35 +611,36 @@ int wc_ed448_import_public(const byte* in, word32 inLen, ed448_key* key)
...
-            ret = ge448_compress_key(key->p, in+1, in+1+ED448_PUB_KEY_SIZE);
-            if (ret == 0)
-                key->pubKeySet = 1;
-        }
-        else if (inLen == ED448_PUB_KEY_SIZE) {
-            /* if not specified compressed or uncompressed check key size
-             * if key size is equal to compressed key size copy in key */
-            XMEMCPY(key->p, in, ED448_PUB_KEY_SIZE);
+    if (inLen < ED448_PUB_KEY_SIZE)
+        return BAD_FUNC_ARG;
```
Comment (SparkiDev) [inline wolfcrypt/src/ed448.c:619]: One exit point per function in this file.

File: wolfcrypt/src/ed448.c [hunk 3]
```diff
@@ -611,35 +611,36 @@ int wc_ed448_import_public(const byte* in, word32 inLen, ed448_key* key)
     int ret = 0;
 
     /* sanity check on arguments */
-    if ((in == NULL) || (key == NULL)) {
-        ret = BAD_FUNC_ARG;
+    if (in == NULL || key == NULL) {
```
Comment (SparkiDev) [inline wolfcrypt/src/ed448.c:614]: Bracket each equality check in this file.

**Comment by SparkiDev** [comment]
https://github.com/wolfSSL/wolfssl/pull/3206

Thanks for showing us the fix!

**Pull request closed without merging**